docs/source/setup.rst

```diff
@@ -0,0 +1,534 @@
+    $ pip3 install "git+https://github.com/someengineering/cloudkeeper.git@main#egg=cksh&subdirectory=cksh"
+
+Usage
+-----
+:ref:`setup-cksh` uses the following command line arguments:
+
+.. code-block:: bash
+    :caption: :ref:`setup-cksh` parameters
+
+    -h, --help            show this help message and exit
+    --ckcore-uri CKCORE_URI
+                        ckcore URI (default: http://localhost:8900)
+    --ckcore-ws-uri CKCORE_WS_URI
+                        ckcore Websocket URI (default: ws://localhost:8900)
+    --ckcore-graph CKCORE_GRAPH
+                        ckcore graph name (default: ck)
+    --stdin               Read from STDIN instead of opening a shell
+    --verbose, -v         Verbose logging
+    --logfile LOGFILE     Logfile to log into
+
+ENV Prefix: ``CKSH_``
+
+Every CLI arg can also be specified using ENV variables.
+
+For instance ``--ckcore-uri http://foobar.tld:8900`` would become ``CKSH_CKCORE_URI=http://foobar.tld:8900``.
+
+Run cksh
+----------
+Now you can connect :ref:`setup-cksh` to the previous setup :ref:`setup-ckcore`.
+Please match your parameter values to reflect your environment while running :ref:`setup-ckcore`.
+
+We add the ``--verbose`` on first start to get used to what is happening exactly.
+You can skip this argument later to reduce log output volume when all components are set up.
+
+Add ``--ckcore-uri`` and ``--ckcore-ws-uri`` if :ref:`setup-ckcore` is running on another instance or port.
+Add ``--ckcore-graph`` if you defined another name of the graph for :ref:`setup-ckworker`
+
+.. code-block:: bash
+    :caption: Run cksh
+
+    $ cksh --verbose \
+      --ckcore-uri ${CKCORE_URI:-http://127.0.0.1:8900} \
+      --ckcore-ws-uri ${CKCORE_WS_URI:-ws://127.0.0.1:8900} \
+      --ckcore-graph ${CKCORE_GRAPH:-ck}
+
+.. code-block:: console
+    :caption: Successful launch log output
+
+    20:25:11 [INFO] Starting up... [core.__main__]
+    20:25:11 [DEBUG] Using selector: KqueueSelector [asyncio]
+    20:25:11 [INFO] Create ArangoHTTPClient with timeout=900 and verify=True [core.db.arangodb_extensions]
+    20:25:11 [INFO] No authentication requested. [core.web.auth]
+    20:25:11 [DEBUG] Starting new HTTP connection (1): localhost:8529 [urllib3.connectionpool]
+    20:25:11 [DEBUG] http://localhost:8529 "GET /_db/cloudkeeper/_api/collection HTTP/1.1" 200 1845 [urllib3.connectionpool]
+    [...]
+    20:25:11 [INFO] Found graph: ck [core.db.db_access]
+    [...]
+    20:25:11 [INFO] Initialization done. Starting API. [core.__main__]
+    20:25:11 [INFO] Listener task_handler added to following queues: ['*'] [core.event_bus]
+    20:25:11 [DEBUG] Looking for jobs to run [apscheduler.scheduler]
+    20:25:11 [DEBUG] Next wakeup is due at 2021-10-04 19:00:00+00:00 (in 2088.660527 seconds) [apscheduler.scheduler]
+    ======== Running on http://localhost:8900 ========
+    (Press CTRL+C to quit)
+
+You made it!
```

Comment:
We should add some info about using a pre-shared key to secure the installation somewhat.